miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt

```diff
@@ -233,13 +232,13 @@ internal class MiniAppSecureStorage(private val activity: Activity) {
                     if (storedItems.containsKey(key)) {
                         onSuccess(storedItems[key] ?: "")
                     } else {
-                        onFailed(MiniAppSecureStorageError.secureStorageIOError)
+                        onSuccess("null")
```

Comment:
Thanks for PR @munir-rakuten , I thinking to make changes in iOS to return `secureStorageIOError` if no key is found. @corycaywood your thoughts ?

Reply:
@rleojoseph I think it should return `null` in this case. That matches how the browser's LocalStorage is working. 
The user will know that the key doesn't exist if `null` is returned, so I think there's no need to return an error.